greenplumpython/builtin/function.py

```diff
@@ -1,53 +1,53 @@
-from typing import Any, Iterable, Optional, Union
+from typing import Any, Optional
 
 from greenplumpython.db import Database
-from greenplumpython.expr import Expr
-from greenplumpython.func import FunctionExpr
+from greenplumpython.func import FunctionExpr, aggregate, function
 from greenplumpython.group import TableRowGroup
 
 
 def count(
     arg: Optional[Any] = None,
     group_by: Optional[TableRowGroup] = None,
-    as_name: Optional[str] = None,
     db: Optional[Database] = None,
 ) -> FunctionExpr:
     if arg is None:
-        return FunctionExpr("count", ["*"], group_by=group_by, as_name=as_name, db=db)
-    return FunctionExpr("count", [arg], group_by=group_by, as_name=as_name, db=db)
+        return FunctionExpr(aggregate("count"), ("*",), group_by=group_by, db=db)
+    return FunctionExpr(aggregate("count"), (arg,), group_by=group_by, db=db)
 
 
 def min(
     arg: Any,
     group_by: Optional[TableRowGroup] = None,
-    as_name: Optional[str] = None,
     db: Optional[Database] = None,
 ) -> FunctionExpr:
-    return FunctionExpr("min", [arg], group_by=group_by, as_name=as_name, db=db)
+    return FunctionExpr(aggregate("min"), (arg,), group_by=group_by, db=db)
 
 
 def max(
     arg: Any,
     group_by: Optional[TableRowGroup] = None,
-    as_name: Optional[str] = None,
     db: Optional[Database] = None,
 ) -> FunctionExpr:
-    return FunctionExpr("max", [arg], group_by=group_by, as_name=as_name, db=db)
+    return FunctionExpr(aggregate(name="max"), (arg,), group_by=group_by, db=db)
```

Comment:
```suggestion
    return FunctionExpr(aggregate("max"), (arg,), group_by=group_by, db=db)
```

Can we omit the argument's name here?